widget/embedded/src/store/slices/data.ts

```diff
@@ -29,6 +29,9 @@ export type FetchStatus = 'loading' | 'success' | 'failed';
 export interface DataSlice {
   _blockchainsMapByName: Map<string, BlockchainMeta>;
   _tokensMapByTokenHash: Map<TokenHash, Token>;
+  _tokensMapByBlockchainName: Record<string, Record<TokenHash, Token>>;
+  _supportedSourceTokens: Map<TokenHash, Token>;
```

Comment:
We currently have this Map for the token,
I think it is better to use Array for supported tokens.